src/librustdoc/html/render/print_item.rs

```diff
@@ -1046,7 +1046,8 @@ fn item_enum(w: &mut Buffer, cx: &Context<'_>, it: &clean::Item, e: &clean::Enum
                 e.generics.print(cx),
                 print_where_clause(&e.generics, cx, 0, true),
             );
-            if e.variants.is_empty() && !e.variants_stripped {
+            let variants_stripped = e.has_stripped_fields();
```

Comment:
I think this code is supposed to check for stripped _variants_, not stripped _fields_.

Reply:
I'll rename this method then.